### ISSUE TITLE
Some Modifications on the Training Code

### ISSUE THREAD

**Issue by ahkarami** [miraclewkf/multilabel-MXNet#4]
Dear @miraclewkf,
As I have investigated your nice training code, I have understand that the CNN model will be _fine-tuned_ on the target multi-label data set. Would you please guide me to modify your code in order that the CNN model _train from scratch_ on the target data set? 
Just as another question, If one wants to change your code in a way that the result model save for example after each 10 epochs (instead of after each epoch), how must changed your training code?

**Comment by jyuan-git** [comment]
You just need to add this at the end of function.
`mx.callback.do_checkpoint(prefix,period=10)`

**Comment by ahkarami** [comment]
Dear @jyuan1118,
Thank you very much for your time and response. Would you please guide me to modify the training code in a way that the CNN model _train from scratch_ on the target data set?
